src/style.css

```diff
@@ -2,6 +2,10 @@
   font-family: "Josefin Sans", sans-serif;
 }
 
+body {
+  background-color: aquamarine;
+}
+
 h1 {
```

Comment:
- [ ] Your app styling is expected to follow the wireframe. The inputs overlap with the scores list. In the wireframe, there is some space between the scores listing and the scores inputs - as if they belong in different halves of the page. Kindly implement that.

  Your styling | Wireframe
  :---:|:---:
  ![ld-2](https://user-images.githubusercontent.com/54848363/181294159-802e061d-e98e-458a-b0f9-e6d31c3a02e2.jpg) | ![ld-3](https://user-images.githubusercontent.com/54848363/181294153-49b100de-8759-431b-8140-249548b6dd7f.jpg)

Reply:
_I think for the npm start the installation process is required in your computer, below is my npm start_

![Screenshot 2022-07-27 183318](https://user-images.githubusercontent.com/93027816/181301936-7b8653cd-39e9-42ea-af3e-d0aa65937fb1.png)

_The add scores is not overlaping in a 100% zoom_

![Screenshot (33)](https://user-images.githubusercontent.com/93027816/181302230-b0eaa01e-d130-40e5-8ba1-a0e7807ec76b.png)